messaging/src/main/java/org/axonframework/eventhandling/TrackingEventProcessor.java

```diff
@@ -648,11 +739,195 @@ public boolean isReplaying() {
 
         @Override
         public TrackingToken getTrackingToken() {
-            if (trackingToken instanceof ReplayToken) {
-                return ((ReplayToken) trackingToken).unwrap();
-            }
+            return WrappedToken.unwrapLowerBound(trackingToken);
+        }
+
+        private TrackingToken getInternalTrackingToken() {
             return trackingToken;
         }
+
+        public TrackerStatus[] split() {
```

Comment:
I think this function needs some javadoc. What's the goal of this `split`? What does it bring to the `TrackerStatus`? Should it be part of the interface too?